testsupport/wait/host.go

```diff
@@ -271,7 +271,7 @@ func (a *HostAwaitility) printMasterUserRecordWaitCriterionDiffs(actual *toolcha
 		}
 	}
 	// also include other resources relevant in the host namespace, to help troubleshooting
-	buf.WriteString(a.sprintAllResources())
+	// buf.WriteString(a.sprintAllResources())
```

Comment:
feel free to remove it and replace with something shorter like: 
```suggestion
	a.listAndPrint("UserSignups", a.Namespace, &toolchainv1alpha1.UserSignupList{})
	a.listAndPrint("MasterUserRecords", a.Namespace, &toolchainv1alpha1.MasterUserRecordList{})
```
so it prints only the related resoruces.